packages/react-components/react-tabs/package.json

```diff
@@ -28,22 +28,22 @@
     "@fluentui/eslint-plugin": "*",
     "@fluentui/react-conformance": "*",
     "@fluentui/react-conformance-griffel": "9.0.0-beta.8",
-    "@fluentui/react-context-selector": "9.0.0-rc.10",
```

Comment:
No, the dependency was, for whatever reason, added twice here, once in `devDependencies` and once in `dependencies`. I'm simply removing the one in `devDependencies` to avoid unnecessary duplication.